docs/Berger_POPC_test_case/comp_JMelcrProg/calcOrderParametersPYTHON3.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env python
```

Comment:
We could remove this file since it's downloaded in the `launch.sh`

Reply:
Well, I wouldn't. If it changes in the future, at least we have the version I used. OK with you?